ngraph/test/backend/gelu.in.cpp

```diff
@@ -89,3 +94,39 @@ NGRAPH_TEST(${BACKEND_NAME}, gelu_f64)
     handle->call_with_validate({result}, {a});
     EXPECT_TRUE(test::all_close_f(input, read_vector<double>(result)));
 }
```

Comment:
Please rewrite old test cases to also use TestCase class. It'll be much cleaner this way.

Reply:
One of the old tests is just gelu_f32 forward pass which I added with TestCase class, the second one is gelu_f64, but from 
what I know f64 is not supported or at least I am getting error telling me that. So should I remove the old test instances?